scripts/xtensa-build-zephyr.py

```diff
@@ -743,6 +744,12 @@ def build_platforms():
 			TOOLCHAIN_VER = platform_dict["XTENSA_TOOLS_VERSION"]
 			XTENSA_CORE = platform_dict["XTENSA_CORE"]
 			platf_build_environ["TOOLCHAIN_VER"] = TOOLCHAIN_VER
+			hifi_version_index = XTENSA_CORE.lower().find("hifi")
+			if hifi_version_index < 0:
+				HIFI_VERSION = "generic"
+			else:
+				HIFI_VERSION = XTENSA_CORE[hifi_version_index+4]
```

Comment:
may be  we can initialize the HIFI_VERSION = "generic"  and  just 
`			if hifi_version_index >= 0:
				HIFI_VERSION = XTENSA_CORE[hifi_version_index+4]
`

Reply:
I don't find >= in python....